tests/nest_tests/stdp_neuromod_test.py

```diff
@@ -72,6 +77,8 @@ def setUp(self):
                              codegen_opts={"neuron_parent_class": "ArchivingNode",
                                            "neuron_parent_class_include": "archiving_node.h"})
 
+    @pytest.mark.skipif(nest_version.startswith("v2"),
```

Comment:
I saw some changes to the `SynapseHeader` template for volume transmitter to make it compatible with NEST version 2. Since this test does not support NEST 2, does this mean that those changes can be removed?

Reply:
The compatibility could still be used by NESTML users even though it's not unit tested right now.